src/connect_wrap.h

```diff
@@ -0,0 +1,26 @@
+#ifndef SRC_CONNECT_WRAP_H_
+#define SRC_CONNECT_WRAP_H_
+
+#if defined(NODE_WANT_INTERNALS) && NODE_WANT_INTERNALS
+
+#include "env.h"
+#include "req-wrap.h"
+#include "async-wrap.h"
+#include "v8.h"
+
+namespace node {
+
+class ConnectWrap : public ReqWrap<uv_connect_t> {
+ public:
+  ConnectWrap(Environment* env,
+      v8::Local<v8::Object> req_wrap_obj,
+      AsyncWrap::ProviderType provider);
```

Comment:
Will do. Sorry, I was a little confused before with when to line up and when not to. Think I've got it now though, thanks for clearing that up.